### PR TITLE
Project 1: List Structure

### PR DESCRIPTION
**Pull request by eri8-9**
# Project 1: List structure.
- Set up a new project with webpack.
- Create an index.html.
- Create an index.js file and set an array of some simple to do tasks (array of objects).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,54 @@
...
+  - Persistent
+
+## Getting Started
+
+### Prerequisites
+  - You should have VS Code installed or have the source-code editor of your preference.
+### Setup
+  - Paste the link of the repo (you can find it in the tab "Code").
+### Install
+  - Use terminal to install it. You can use git clone follow by link of the repo.
```
Comment (mricanho) [inline README.md:32]: - [ ] You need to be more specific in this sections, always write the instructions from a non-developer perspective

**Review by mricanho — CHANGES_REQUESTED** [review]
Hi @eri8-9, 

Good job so far! 🔥🔥 Your JS code it's good, as well as your webpack config, just a minor detail on your README file

There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

you can do it

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @eri8-9😃, 

You met all the requirements on this project 💯 Keep rocketing 🚀
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by hemant-soni-vst-au4 — COMMENTED** [review]
Hello @eri8-9 👋
While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that is having the previous week's PR. Please try to create a new PR as you are repeating this week.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers,
Hemant Soni
Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **Does not count** into the code reviews limit._

------

**Pull request merged**